packages/showcase/src/app/components/tooltip/metadata.ts

```diff
@@ -0,0 +1,158 @@
+import { ComponentMetadata } from "../../../common/types";
+import { chameleonImportType } from "../../../services/template-language/create-template";
+
+import { CSS } from "../../../common/html";
+
+const ICON_SRC =
+  'getIconPath({ category: "system", name: "refresh", colorType: "on-primary" })';
+
+export const tooltipMetadata = {
+  title: "Tooltip",
+  description:
+    "A tooltip in web development is a small, interactive pop-up that appears when a user hovers over or focuses on an element. It provides additional information or context without cluttering the interface, offering users quick insights about features or actions. Tooltips are commonly used to enhance usability by clarifying unclear elements or providing guidance.",
+  codeSnippets: {
+    blockStart: {
+      linkId: "block-start",
+      title: "1. Block Start",
+      imports: [
+        'import { getIconPath } from "@genexus/mercury/assets-manager.js";'
+      ],
+      variables: [
+        {
+          name: "ICON_SRC",
+          value: ICON_SRC
+        }
+      ],
+      template: {
+        tag: "button",
+        class: "button-primary button-icon-only",
+        properties: [{ name: "type", value: "button" }],
+        children: [
+          {
+            tag: "ch-image",
+            class: "icon-md",
+            properties: [{ name: "src", value: "ICON_SRC", variable: true }]
+          },
+          {
+            tag: CSS.TOOLTIP.TAG,
+            class: CSS.TOOLTIP.SELECTOR,
+            properties: [
+              { name: "actionElement", value: "null", variable: true },
+              { name: "blockAlign", value: "outside-start" },
+              { name: "inlineAlign", value: "center" }
+            ],
+            children: ["Refresh to update your settings"]
+          }
+        ]
+      }
+    },
```

Comment:
Buttons must always have a label (visible or `aria-label`) to describe their content. You can add an `aria-label` in the button or use the `actionElementAccessibleName` property of the ch-tooltip:

https://github.com/genexuslabs/chameleon-controls-library/tree/main/src/components/tooltip#ch-tooltip